.github/workflows/push_trigger.yml

```diff
@@ -836,14 +836,14 @@ jobs:
           echo VERSION=$VERSION
           docker tag $SERVICE_NAME $IMAGE_ID:$VERSION
           docker push $IMAGE_ID:$VERSION
-  docker-registration-processor-landing-zone-utility:
+  docker-registration-processor-landing-zone:
     needs: build
 
     runs-on: ubuntu-latest
     env:
       NAMESPACE: mosipdev
```

Comment:
Please use secrets variable instead of mosipdev

Reply:
Updated can you pls check.